jar-infer/nullaway-integration-test/build.gradle

```diff
@@ -15,9 +15,6 @@
  */
 plugins {
   id "java"
-  // For code coverage:
-  id 'jacoco'
-  id 'com.github.kt3k.coveralls'
```

Comment:
Coverage plugin has [multi-project support](https://github.com/nbaztec/coveralls-jacoco-gradle-plugin#multi-project-support---android). But:
a) It's a bit convoluted
b) We have never had proper coverage support for JarInfer before (this bit above is long broken)
c) We mostly care about coverage for core NullAway

That said, it's worth having JarInfer coverage in the future as well, just not high priority.